src/server/event_bus.c

```
@@ -0,0 +1,265 @@
+}
+
+/* Apply all pending removals accumulated during a fire(). */
+static void flush_removals(obc_event_entry_t *e)
+{
+    for (int r = 0; r < e->remove_count; r++)
+        remove_sub(e, e->remove_pending[r]);
+    e->remove_count = 0;
+}
+
+/* Apply all pending additions accumulated during a fire(). */
+static void flush_additions(obc_event_entry_t *e)
```

Clang (14.0.6)

[warning] parameter name 'e' is too short, expected at least 3 characters

(readability-identifier-length)